Feature/gui master

TheWinczi opened the pull request:
Better GUI implementation. Drawing the best paths, generating diagrams, more learning settings.

TheWinczi committed:
GUI improvements - drawing paths, more world and strategies options

TheWinczi committed:
GUI code refactoring

TheWinczi committed:
Cleaning world method was added

Waffele commented:
Przydało by się aby dało się rozpoznać czy agent jeszcze pracuje czy nie. Proponuję po kliknięciu start albo start learning wyczyścić tą mini konsolkę na dole.

Waffele commented:
I też mam prośbę abyś sparametryzował wielkość wektora środowiska (3x3, 5x5 itd) żeby to można było w pliku ze stałymi zmienić

TheWinczi committed:
Some features was added
1. Point's environment vector as constant parameter
2. Clearing text field when agent is start learning

Waffele commented:
no i kozak
